apps/google-analytics-4/frontend/src/helpers/DateRangeHelpers/DateRangeHelpers.spec.ts

```diff
@@ -1,9 +1,11 @@
 import { DateRangeType } from '@/types';
 import getRangeDates, { RANGE_OPTIONS } from './DateRangeHelpers';
 
+const DAY_IN_MS = 1000 * 3600 * 24;
```

Comment:
_maybe_ better to have the whole expression like: `1000 * 60 * 60 * 24`? not a blocker :)